packages/expo-react-native-adapter/src/__tests__/NativeViewManagerAdapter-test.tsx

```diff
@@ -19,8 +19,15 @@ jest.mock('react-native', () => {
 
 describe('requireNativeViewManager', () => {
   it(`sets the "displayName" of the native component`, () => {
-    const TestView = requireNativeViewManager('ExpoTestView');
-    expect(TestView.displayName).toBe('Adapter<ExpoTestView>');
+    // USING REACT INTERNALS HERE!
+    // `getComponentName()` isn't exported from React, so we can't test
```

Comment:
Run https://marketplace.visualstudio.com/items?itemName=stkb.rewrap or a similar plugin on this comment so it wraps at the column boundary (100 columns).